src/Intro.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+export default class Settings extends React.Component {

```

Comment:
Please name it according to the file name. You already have `Settings` component